src/js/wasm.js-post.js

```diff
@@ -1,351 +0,0 @@
-/*
```

Comment:
this file is removed because it moved repos a while ago, the copy here was forgotten. removing it now because it contains stuff that refers to the old ffi mechanism, so it could potentially confuse people grepping